### PR TITLE
Refactor Ruby scl role to be generic

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

File: roles/ruby_scl/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    - "{{ ruby_scl_version }}"
+    - "{{ ruby_scl_version }}-ruby-devel"
+
+- name: 'Enable SCL at login'
+  blockinfile:
+    dest: /etc/profile.d/enable-{{ ruby_scl_version }}.sh
+    create: yes
+    block: |
+      #!/bin/bash
+      source /opt/rh/{{ ruby_scl_version }}/enable
```
Comment (ekohl) [inline roles/ruby_scl/tasks/main.yml]: The modern way is ```source scl_source enable {{ ruby-scl_version }}``` which also sets ```X_SCLS``` for you

Comment (ehelms) [reply roles/ruby_scl/tasks/main.yml]: Good catch!

**Comment by ehelms** [comment]
Updated

File: roles/ruby_scl/tasks/main.yml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    - "{{ ruby_scl_version }}-ruby-devel"
+
+- name: 'Enable SCL at login'
+  blockinfile:
+    dest: /etc/profile.d/enable-{{ ruby_scl_version }}.sh
+    create: yes
+    block: |
+      #!/bin/bash
+      source scl_source enable {{ ruby_scl_version }}
+
```
Comment (ekohl) [inline roles/ruby_scl/tasks/main.yml]: Nit: could you remove the empty line?

**Comment by akofink** [comment]
You missed the `become: true` parts :(

**Pull request merged**